openownership/data-standard
# Schema reference edits

# Overview

**What does this pull request do?** 
 
- updates the schema ref page 
- updates the schema browser page
- fixes issues with the urn resolving in sphinx docson and directives
- fixes other docson issues 

**How can a reviewer test or examine your changes?**

- Run "pip install -r requirements_test.txt". 
- If you aren't starting with a clean repo on disk, you will need to run 'git submodule update --remote'
- Build the docs
- Read through the browser and reference pages. Check all the links

Note that the browser widget has some 'hanging' AllOfs and AnyOfs, and some of the UI isn't great. It just needs to be good enough for now.

Also: on the reference page, there are inconsistencies with sub-headings: some are 'camelCase' and some 'Are Not'. We're not going to resolve that this time.


**Who is best placed to review it?**

@odscrachel 

Closes #528 #546 #682 #557

## Translations

      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

      https://openownership.github.io/bods-dev-handbook/standard_releases.html
